source/agora/consensus/EnrollmentManager.d

```diff
@@ -716,63 +658,48 @@ public class EnrollmentManager
 
     /***************************************************************************
 
-        Get the next block height to reveal a pre-image
-
-        Returns:
-            the next block height to reveal a pre-image. if any problem in
-            getting the value, it returns the MAX ulong value.
+        Get the distance of the pre-image that will be revealed next time
```

Comment:
```suggestion
        Returns:
            Get the distance of the pre-image that will be revealed next time,
            or 0 if there's no preimage to reveal
```

The comment I made about `0` I'm not sure about, you probably know better than me.